homeassistant/components/fritz/switch.py

```diff
@@ -468,11 +467,11 @@ async def _async_fetch_update(self) -> None:
         self._is_available = True
 
         attributes_dict = {
-            "NewInternalClient": "internalIP",
-            "NewInternalPort": "internalPort",
-            "NewExternalPort": "externalPort",
-            "NewProtocol": "protocol",
-            "NewPortMappingDescription": "description",
+            "internal_client": "internalIP",
```

Comment:
It's the dict values here that form the state attribute keys.

Reply:
Indeed... that makes more sense...
I will change it